scripts/create2/deploy.js

```diff
@@ -24,20 +25,25 @@ const usage = () => {
         [gas price] Base fee in gwei. This is a required parameter.
       --priority-fee, -p MINER_TIP
         [gas price] Miner tip in gwei. Default: 2 gwei. This is a required parameter.
-      --gas-limit, -l GAS_LIMIT
+      --gas-limit, -g GAS_LIMIT
```

Comment:
changed this to `-g` so the `-l` flag is consistent to both find-salt and deploy CLI scripts